Extension/src/LanguageServer/client.ts

```diff
@@ -1292,8 +1300,11 @@ export class DefaultClient implements Client {
                     },
                     space:  vscode.workspace.getConfiguration("C_Cpp.vcFormat.space", this.RootUri),
                     wrap:  vscode.workspace.getConfiguration("C_Cpp.vcFormat.wrap", this.RootUri)
-                },
-                tabSize: vscode.workspace.getConfiguration("editor.tabSize", this.RootUri)
+                }
+            },
+            editor: {
+                tabSize: otherSettingsFolder.editorTabSize,
```

Comment:
tabSize has been removed on the native side, as it wasn't needed.  (The tab size is also included in format requests).  So, I think this handling of tabSize can be removed from the TypeScript side.

Reply:
Yeah, I noticed that. I can remove it.